presto-main/src/main/java/com/facebook/presto/util/MergeSortedPages.java

```diff
@@ -153,14 +154,14 @@ private MergeSortedPages() {}
                 int position;
 
                 @Override
-                public ProcessorState<PageWithPosition> process()
+                public ProcessState<PageWithPosition> process()
                 {
                     if (position >= page.getPositionCount()) {
                         memoryContext.close();
-                        return ProcessorState.finished();
+                        return finished();
```

Comment:
Now one can only static import state methods for either `TransformationState/ProcessState`, but that isn't a big issue

Reply:
Yes. Unfortunately. But
- i didn't create artificially different names for the sake of import; that would be misleading, as the two State classes are not purposefulyl similar
- when you implement Transformation or Process as a class (potentially anonymous), you don't need import at all

Reply:
> when you implement Transformation or Process as a class (potentially anonymous), you don't need import at all

Right, nice